### PR TITLE
EPMRPP-37347. Admin. Projects. Single panel.

### PR DESCRIPTION
**Pull request by ElenaRomanchuk**
EPMRPP-37347. Admin. Projects. Single panel.

### CODE REVIEW

File: app/src/pages/admin/projectsPage/project/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { ProjectInfo } from './projectInfo';
```
Comment (AmsterGet) [inline app/src/pages/admin/projectsPage/project/index.js]: Please, follow the our common naming:
`folderName` = `componentFileName`
So please rename `project` to `projectInfo`

File: app/package.json [hunk 2]
```diff
@@ -78,6 +79,7 @@
     "reset-css": "2.2.1",
     "semver-diff": "2.1.0",
     "simplemde": "1.11.2",
+    "storybook-decorators": "^0.3.2",
```
Comment (AmsterGet) [inline app/package.json]: Can you use our decorators from `.storybook/decorators`?

File: app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  };
+
+  updateMenuPosition = () => {};
+
+  render() {
+    const { items, disabled, buttonClassName, tooltip } = this.props;
+    return (
+      <Manager>
+        <div
+          ref={(node) => {
```
Comment (AmsterGet) [inline app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx]: Please use new ref mechanism (React.createRef).

File: app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                        <div
+                          key={item.value}
+                          className={cx('menu-item', {
+                            disabled: item.disabled,
+                            danger: item.type === 'danger',
+                          })}
+                          title={item.title || ''}
+                          onClick={
+                            !item.disabled
+                              ? (e) => {
```
Comment (AmsterGet) [inline app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx]: Please move it to a separate function

File: app/src/pages/admin/projectsPage/project/projectInfo.jsx [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+            <span className={cx('header-stretch-item', 'gray-text')}>{organization}</span>
+            <span className={cx('header-item')}>
+              <DotsMenuButton items={this.getMenuItems()} />
+            </span>
+          </div>
+          <NavLink
+            className={cx('name')}
+            to={{
+              type: PROJECT_PAGE,
+              payload: { projectId: id, projectSection: null },
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/project/projectInfo.jsx]: projectId: projectName

File: app/src/pages/admin/projectsPage/project/projectInfo.jsx [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+            />
+            <StatisticsItem
+              caption={intl.formatMessage(messages.membersQuantity)}
+              value={launchesQuantity}
+            />
+            <NavLink
+              className={cx('statistic-button')}
+              to={{
+                type: PROJECT_DETAILS_PAGE,
+                payload: { projectId: id, projectSection: null },
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/project/projectInfo.jsx]: projectId: projectName

File: app/src/pages/admin/projectsPage/project/projectTooltipIcon/projectTooltipIcon.jsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    children: PropTypes.node,
+  };
+  static defaultProps = {
+    tooltipContent: '',
+    children: '',
+  };
+
+  render() {
+    const { children } = this.props;
+    return <div content={this.props.tooltipContent}>{children}</div>;
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/project/projectTooltipIcon/projectTooltipIcon.jsx]: `div` doesn't have attribute `content`.
Prop `tooltipContent` is used in the decorator component. It's not needed here.

File: app/src/pages/admin/projectsPage/project/statisticsItem/statisticsItem.scss [hunk 8]
```diff
@@ -0,0 +1,21 @@
+.caption {
+  font-family: $FONT-REGULAR;
+  font-size: 12px;
+  font-weight: normal;
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/project/statisticsItem/statisticsItem.scss]: We don't use `font-weight` property. It comes from selected font

File: app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+
+  getMenuItems(items) {
+    return items
+      .filter((item) => !item.hidden)
+      .map(
+        (item) =>
+          item.type === SEPARATOR_ITEM ? (
+            <Separator key={`separator${item.value}`} />
+          ) : (
+            <MenuItem key={item.value} item={item} toggleMenu={() => this.toggleMenu()} />
```
Comment (DiscoElevator) [inline app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx]: `toggleMenu={this.toggleMenu}`

File: app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+
+  componentDidMount() {
+    document.addEventListener('click', this.handleOutsideClick, false);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.handleOutsideClick, false);
+  }
+
+  getMenuItems(items) {
```
Comment (DiscoElevator) [inline app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx]: `renderMenuItems`

File: app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx (hunk 9 above)
Comment (ElenaRomanchuk) [reply app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx]: It will couse loosing contex in toggleMenu

Comment (DiscoElevator) [reply app/src/components/buttons/dotsMenuButton/dotsMenuButton.jsx]: Just change `getMenuItems` (`renderMenuItems`, as in the next comment) from a class method to a class field with arrow function.
`renderMenuItems = (items) => { ... }`

**Pull request merged**